src/main/java/com/nike/cerberus/cli/CerberusRunner.java

```diff
@@ -84,12 +91,19 @@ public void run(String[] args) {
         registerAllCommands();
 
         try {
+
             commander.parse(args);
             configureLogging(cerberusCommand.isDebug());
             final String commandName = commander.getParsedCommand();
             final Command command = commandMap.get(commandName);
 
-            if (command == null) {
+            if(cerberusCommand.isVersion()) {
```

Comment:
Help seemed to be in relation to a specific command. 

Since version doesn't require a command I thought it made more sense to go above the check for usage help. 

Probably worth adding a general --help flag at the cli level rather than just at the command level.